torch/csrc/jit/serialization/export.cpp

```diff
@@ -543,7 +543,8 @@ void GraphEncoder::EncodeValueInfoType(
         std::unordered_map<int64_t, std::string>>& dynamic_axes) {
   auto tensorTypeToONNXType = [&dynamic_axes, n, this](
                                   const TensorTypePtr& t,
-                                  onnx::TypeProto_Tensor* onnx_tensor_type) {
+                                  onnx::TypeProto_Tensor* onnx_tensor_type,
+                                  bool is_sequence_tensor) {
```

Comment:
I think this would be clearer if we renamed the variable something about what the intended effect is. Something like `bool assign_dim_param`.